docs/pages/docs/guides/filters.mdx

```diff
@@ -43,7 +43,7 @@ Keystone provides a wide range of different filters. If we want to find all thos
 
 ```graphql
 {
-  tasks(where: { label: { not: { equals: "Hello" } } }) {
+  tasks(where: { label: { NOT: { equals: "Hello" } } }) {
```

Comment:
This change isn't correct, the previous casing is correct in this case
```suggestion
  tasks(where: { label: { not: { equals: "Hello" } } }) {
```

Reply:
Oh you're right. Wow, that's pretty confusing, no? :/

Reply:
Good catch though, thanks